ceph.spec.in

```diff
@@ -246,6 +246,7 @@ BuildRequires:	pkgconfig
 BuildRequires:  procps
 BuildRequires:	python%{python3_pkgversion}
 BuildRequires:	python%{python3_pkgversion}-devel
+BuildRequires:	python%{python3_pkgversion}-pysaml2
```

Comment:
Is this a modification relevant to the PR

Reply:
sorry that was unintentional, fixed it